dev-setup/ozone/values.yaml

```diff
@@ -0,0 +1,4 @@
+image:
```

Comment:
here you can put down the non-M1 values as a default. as it can serve as a doc of what values should be put here.